enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -1393,43 +1393,43 @@
                   update-id "table.row/update"
                   delete-id "table.row/delete"]
 
-              (testing "using a partially constructed :input"
-                (let [unrelated-table Long/MAX_VALUE
-                      scope           {:table-id unrelated-table}
-                      input           {:table-id @test-table}]
-                  (testing "create"
-                    (is (=? {:status 200
-                             :body   {:parameters [{:id "text" :readonly false}
-                                                   {:id "int" :readonly false}
-                                                   {:id "timestamp" :readonly false}
-                                                   {:id "date" :readonly false}]}}
-                            (req {:action_id create-id
-                                  :scope     scope
-                                  :input     (assoc input :id 1)}))))
-                  (testing "update"
-                    (is (=? {:status 200} (req {:action_id update-id, :scope scope, :input input}))))
-                  (testing "delete"
-                    (is (=? {:status 200} (req {:action_id update-id, :scope scope, :input input}))))))
+              #_(testing "using a partially constructed :input"
```

Comment:
@crisptrutski this rely on the input case we removed, do you think we need it still?